ds701_book/21-Networks-I.qmd

```diff
@@ -253,422 +387,409 @@ nx.draw_networkx(DG,
 alpha = 1.3
-plt.figure(figsize = (8, 6))
+plt.figure(figsize = (7, 5))
 x = np.linspace(pareto.ppf(0.005,alpha), pareto.ppf(0.995,alpha), 100)
 plt.plot(np.log10(x), np.log10(pareto.pdf(x,alpha)),'b-', lw = 5, alpha = 0.6, label='pareto pdf')
 plt.title(r'Pareto PDF.  $\alpha$ = {}'.format(alpha), size=16)
 plt.xlabel('$\log_{10}(x)$', size=14)
-plt.ylabel('$\log_{10}(p(x))$', size=14);
+plt.ylabel('$\log_{10}(p(x))$', size=14)
+plt.show()
 ```
 
-#### Power Law Degree Distributions are Ubiquitous
+## Power Law Degree Distributions are Ubiquitous
 
-<center>
-    
-<img src="figs/L21-newman-powerlaws.png" alt="Figure" width="800px">
-    
-</center>
+![](figs/L21-newman-powerlaws.png){fig-align=center width=40%}
+
+---
 
 > The networks shown are: (a) the collaboration network of mathematicians [182]; (b) citations between 1981 and 1997 to all papers cataloged by the Institute for Scientific Information [351]; (c) a 300 million vertex subset of the World Wide Web, circa 1999 [74]; (d) the Internet at the level of autonomous systems, April 1999 [86]; (e) the power grid of the western United States [416]; (f) the interaction network of proteins in the metabolism of the yeast S. Cerevisiae [212]. 
 >
 > _The structure and function of complex networks,_ M. E. J. Newman
 >
 > https://arxiv.org/abs/cond-mat/0303516
 
-Note that $G(n, p)$ random graphs do __not__ have power-law degree distributions.
+We remark that $G(n, p)$ random graphs do __not__ have power-law degree distributions. 
 
-Their degree distributions have exponential tails (like the power grid example above.)
+Their degree distributions have exponential tails, similar to the power grid example above. Note that the $x$-axis of the power grid example is a linear scale but the $y$-axis is a log scale.
 
-### Clustering
+## Clustering
 
 The next important property of a network to understand is __clustering__.  
 
 In the context of networks, clustering refers to the tendency for groups of nodes to have higher connectivity within the group than the network-wide average.
 
 The simplest measure of local clustering is __clustering coefficient__.
 
-The clustering coefficient is the answer to the question "are my friends themselves friends?"
+The clustering coefficient tells you if the neighbors of a node tend to be neighbors.
 
-In other words, do the neighbors of a node tend to be neighbors?
+---
 
-More specifically, clustering coefficient measures the __probability that two of your neighbors are connected.__ 
+Specifically, clustering coefficient measures the __probability that two of your neighbors are connected.__ 
 
-There are two ways one can measure this:
+There are two ways to measure this, the first is
 
-$$C^{(1)} = \frac{\sum_i \text{number of triangles that include }i}{\sum_i \text{number of pairs of neigbors of }i} $$
+$$
+C^{(1)} = \frac{\sum_i \text{number of triangles that include node } i}{\sum_i \text{number of pairs of neighbors of node }i}. 
```

Comment:
What is a "triangle"?

Reply:
Added a definition for what a triangle is in a network.